src/CopyrighFixer/CopyrighFixer/worker.cpp

```diff
@@ -6,18 +6,39 @@
 //#
 
 #include "config.h"
+#include "configparser.h"
 #include "worker.h"
 #include <quasarapp.h>
 
 
-
 namespace CopyrighFixer {
-Worker::Worker() {
 
+Worker::Worker(
+        Config *conf,
+        ConfigParser *confParser,
+        Signer *subscriber) {
+    conf_ = conf ?: new Config;
+    confParser_ = confParser ?: new ConfigParser;
+    subscriber_ = subscriber ?: new Signer;
```

Comment:
the worker object must be initialized without arguments.
```suggestion
Worker::Worker() {
    conf_ = new Config;
    confParser_ =  new ConfigParser;
    subscriber_ = new Signer;
```